### PR TITLE
On builds outside Windows, build native test components by default

### PR DESCRIPTION
**Pull request by kouvel**
Not removing the 'includetests' parameter for now, as dependencies on it have been taken.

### CODE REVIEW

**Comment by kouvel** [comment]
@sergiy-k @mmitche

**Comment by mmitche** [comment]
@dotnet-bot test this please (FreeBSD machine was horked)

File: build.sh [hunk 1]
```diff
@@ -12,7 +12,8 @@ usage()
     echo "cross - optional argument to signify cross compilation,"
     echo "      - will use ROOTFS_DIR environment variable if set."
     echo "skipmscorlib - do not build mscorlib.dll even if mono is installed."
-    echo "includetests - build the tests in the 'tests' subdirectory as well."
+    echo "includetests - build the tests in the 'tests' subdirectory as well (on by default)."
```
Comment (mmitche) [inline build.sh]: Since this no longer does anything, we should remove it.

Comment (kouvel) [inline build.sh]: Right, will do

**Comment by mmitche** [comment]
LGTM

**Pull request merged**